Lib/AspNet/Microsoft.WindowsAzure.Storage/project.json

```diff
@@ -1,20 +1,18 @@
 ﻿{
-    "version": "7.1.3.0",
+    "version": "7.1.4.0",
     "authors": [ "Microsoft Corporation" ],
     "description": "Azure Storage SDK for CoreCLR Preview",
     "dependencies": {
         "Microsoft.Data.OData": "5.6.4",
         "Microsoft.Data.Services.Client": "5.6.4",
         "System.Spatial": "5.6.4",
-        "Newtonsoft.Json": "6.0.8"
+        "Newtonsoft.Json": "9.0.1"
     },
 
     "frameworks": {
-        "netstandard1.6": {
+        "netstandard1.3": {
             "dependencies": {
-                "NETStandard.Library": "1.6.0",
-                "System.Dynamic.Runtime": "4.0.11",
-                "System.Security.Cryptography.Algorithms": "4.2.0"
+                "NETStandard.Library": "1.6.0"
```

Comment:
As a comment for anyone who is curious, we received feedback about not including the all-up library package and instead including each dependency individually, but the netstandard team's guidance is to use this all-up package. This is to eliminate the chance of getting incompatibilities between subpackages. They're intended to be included together and there will be some documentation being published imminently recommending this approach.